src/main/java/net/dv8tion/jda/api/events/thread/member/ThreadMemberJoinEvent.java

```diff
@@ -23,21 +23,31 @@
 
 import javax.annotation.Nonnull;
 
-//TODO-v5: Docs
+/**
+ * Indicates that a {@link Member GuildMember} joined a {@link ThreadChannel}
+ */
 public class ThreadMemberJoinEvent extends GenericThreadMemberEvent
```

Comment:
We need to document when you can actually receive these events.
I believe we can only get them if you are yourself joined to the thread.
You might also need certain intents. I can't remember.

Reply:
I'll figure this out :)